ttnn/cpp/ttnn/operations/moreh/moreh_dot/device/moreh_dot_program_factory.cpp

```diff
@@ -67,56 +65,57 @@ MorehDotOperation::SingleCore::cached_program_t MorehDotOperation::SingleCore::c
     std::vector<uint32_t> reader_compile_time_args = {
         (std::uint32_t)tt::operations::primary::is_dram(src0_buffer),
         (std::uint32_t)tt::operations::primary::is_dram(src1_buffer),
-        *reinterpret_cast<uint32_t *>(&scaler)};
+        *reinterpret_cast<uint32_t*>(&scaler)};
 
-    std::vector<uint32_t> writer_compile_time_args = {(std::uint32_t)CB::c_out0, (std::uint32_t)tt::operations::primary::is_dram(dst_buffer)};
-    const auto reader_kernel_file = "ttnn/cpp/ttnn/operations/moreh/moreh_dot_op/device/kernels/reader_moreh_dot.cpp";
-    const auto writer_kernel_file = "ttnn/cpp/ttnn/operations/moreh/moreh_dot_op/device/kernels/writer_moreh_dot.cpp";
+    std::vector<uint32_t> writer_compile_time_args = {
+        (std::uint32_t)CB::c_out0, (std::uint32_t)tt::operations::primary::is_dram(dst_buffer)};
+    const auto reader_kernel_file = "ttnn/cpp/ttnn/operations/moreh/moreh_dot/device/kernels/reader_moreh_dot.cpp";
+    const auto writer_kernel_file = "ttnn/cpp/ttnn/operations/moreh/moreh_dot/device/kernels/writer_moreh_dot.cpp";
 
-    const auto reader_kernel_id = tt::operations::primary::CreateReadKernel(program, reader_kernel_file, core, reader_compile_time_args);
-    const auto writer_kernel_id = tt::operations::primary::CreateWriteKernel(program, writer_kernel_file, core, writer_compile_time_args);
+    const auto reader_kernel_id =
+        tt::operations::primary::CreateReadKernel(program, reader_kernel_file, core, reader_compile_time_args);
+    const auto writer_kernel_id =
```

Comment:
As `compute_kernel_config` defined.
Need specify `math_fidelity, math_approx_mode, fp32_dest_acc_en` by `compute_kernel_config`

Reply:
In other cases, I don't see anyone defining `math_fidelity, math_approx_mode, fp32_dest_acc_en` for read kernel or write kernel